localization/languages/tr.json

```diff
@@ -267,11 +267,11 @@
     "passwordCaptureSavePassword": "% için parola kaydedilsin mi?",
     "passwordCaptureSave": "Kaydet",
     "passwordCaptureDontSave": "Kaydetme",
-    "passwordCaptureNeverSave": null, //missing translation
+    "passwordCaptureNeverSave": "% için parolaları asla kaydetme", 
```

Comment:
The English version of this is just "never save": https://github.com/minbrowser/min/blob/master/localization/languages/en-US.json#L270

Reply:
Yes, this is locatilazed version. It is being used like that in Turkish.

Reply:
I just want to make sure the Turkish version has the same meaning as the English one. The Turkish one here has a % sign in it, and Google Translate says it means "Never save passwords for %". the original English is just "Never save", without "passwords" or a site name.

It's used here:
<img width="1552" alt="Screen Shot 2021-09-11 at 6 10 45 PM" src="https://user-images.githubusercontent.com/10314059/132964214-a1051ad2-5056-48ed-b310-a25d1a8c09e6.png">

So it should be fairly short.

Reply:
I'm sorry, I didn't know I shouldn't have used the site name. I'll shorten it, edit it, and update it again